### DISCUSSION TITLE
Anyway to change unicode to normal text in saved images?

### DISCUSSION THREAD

**Discussion by Joeyd348039** [vladmandic/sdnext#1675] [General]
Is there anyway to change the way image info is saved? The unicode is annoying due to the spaces it puts in between every single letter.

**Comment by vladmandic** [comment]
Metadata? No. But why would you care about internal encoding of metadata inside the file?

**Reply by Tillerz** [reply]
I think he's as lazy as me and not opening the image in a viewer that shows you the metadata but just opens the image in a text viewer like MidnightCommander, where you see all the binary garbage, but also the meta text as plain iso-... text, and in THAT kind of view, there's spaces everywhere. ;)

![screenshot](https://i.gyazo.com/c81032667b13da9c727e5147fb360a1a.png)

**Reply by vladmandic** [reply]
yeah, metadata is binary. the fact you want to see ascii text in something that is not parsing it at all is not a valid use case.
